src/utils.js

```diff
@@ -6,15 +6,31 @@ import fs from 'node:fs';
 import clone from 'clone';
 import glyphCompose from '@mapbox/glyph-pbf-composite';
 
-export const getPublicUrl = (publicUrl, req) =>
-  publicUrl || `${req.protocol}://${req.headers.host}/`;
+const getUrlObject = (urlString, req) => {
```

Comment:
Do we need to pass urlString here. can we just construct this inside the getUrlObject function from the req parameter? seems like urlString is just `${req.protocol}://${req.headers.host}/`

Reply:
Yes that makes sense. Changed it https://github.com/maptiler/tileserver-gl/pull/733/commits/69b8bcc171361413e7c92174105b6d34899fc75b